ui/src/main/webapp/src/app/project/migration-project-form.component.html

```diff
@@ -17,32 +42,23 @@
             <input #projectTitleControl="ngModel"
                    name="projectTitle"
                    id="idProjectTitle"
+                   type="text"
+                   placeholder="my-project"
+                   required pattern="\s*\w[- \w]+\w\s*" minlength="3" maxlength="120"
                    [(ngModel)]="model.title"
-                   required
                    wuProjectNameNotUsed
-                   pattern="\s*\w[- \w]+\w\s*"
-                   minlength="3"
-                   maxlength="120"
                    [project]="model"
-                   type="text"
-                   placeholder="my-project"
-                   class="form-control">
-            <p class="help-block" i18n="hint">A unique name for the project</p>
-            <span [class.hidden]="!titleIsDuplicated(projectTitleControl)" class="help-block" i18n="valid. error"> The entered name is already in use. </span>
-            <span [class.hidden]="!hasError(projectTitleControl, 'pattern') || hasError(projectTitleControl, 'minlength') || hasError(projectTitleControl, 'maxlength')" class="help-block" i18n="valid. error pattern">
-                The project name must contain only alphanumeric characters.
-            </span>
-            <span [class.hidden]="!hasError(projectTitleControl, 'minlength')" class="help-block" i18n="valid. error min length">
-                The project name must contain at least 3 characters.
-            </span>
-            <span [class.hidden]="!hasError(projectTitleControl, 'maxlength')" class="help-block" i18n="valid. error max length">
-                The project name must contain fewer than 120 characters.
-            </span>
+                   class="form-control"
+                   (blur)="$event.target.value = $event.target.value.trim();"
+            >
+            <div i18n="hint">A unique name for the project</div>
+            <div [class.hidden]="!titleIsDuplicated(projectTitleControl)" class="help-block" i18n="valid. error"> The entered name is already in use. </div>
+            <div class="error-msg-invalid help-block" i18n="valid. error"> The name must consist of 3 to 120 alphanumeric characters and spaces. </div>
         </div>
 
         <div class="form-group">
             <label class="control-label" for="idDescription">Description</label>
-            <textarea name="description" id="idDescription" name="idDescription" class="form-control" i18n="placeholder" placeholder="A short description of the project." [rows]="getDescriptionHeight()"
+            <textarea name="description" id="idDescription" name="idDescription" class="form-control" maxlength="4095" i18n="placeholder" placeholder="A short description of the project." [rows]="getDescriptionHeight()"
```

Comment:
Interesting... I wasn't expecting any changes on this field at all. I guess that Marek broke it in this commit:
https://github.com/windup/windup-web/commit/7cb552eb1c7054beb29e6e595bd32d95673111c7

Its probably good to bring it back.